docs/GettingStarted/configuration/ExporterDeploytime.md

```diff
@@ -2,28 +2,71 @@
+## Deploy Time Exporter configuration options
 
-This exporter provides several configuration options, passed via `pelorus-config` and `deploytime-config` variables. User may define own ConfigMaps and pass to the committime exporter in a similar way.
+This is the list of options that can be applied to `env_from_secrets`, `env_from_configmaps` and `extraEnv` section of a Failure time exporter.
 
-| Variable                  | Required | Explanation                                                                                                       | Default Value                 |
-|---------------------------|----------|-------------------------------------------------------------------------------------------------------------------|-------------------------------|
-| `LOG_LEVEL`               | no       | Set the log level. One of `DEBUG`, `INFO`, `WARNING`, `ERROR`                                                     | `INFO`                        |
-| `APP_LABEL`               | no       | Changes the label key used to identify applications                                                               | `app.kubernetes.io/name`      |
-| `PROD_LABEL`              | no       | Changes the label key used to identify namespaces that are considered production environments.                    | unset; matches all namespaces |
-| `NAMESPACES`              | no       | Restricts the set of namespaces from which metrics will be collected. ex: `myapp-ns-dev,otherapp-ci`              | unset; scans all namespaces   |
-| `PELORUS_DEFAULT_KEYWORD` | no       | ConfigMap default keyword. If specified it's used in other data values to indicate "Default Value" should be used | `default`                     |
+| Variable | Required | Default Value |
+|----------|----------|---------------|
+| [LOG_LEVEL](#log_level) | no | `INFO` |
+| [APP_LABEL](#app_label) | no | `app.kubernetes.io/name` |
+| [NAMESPACES](#namespaces) | no | - |
+| [PROD_LABEL](#prod_label) | no | - |
+| [PELORUS_DEFAULT_KEYWORD](#pelorus_default_keyword) | no | `default` |
+
+###### LOG_LEVEL
+
+- **Required:** no
+    - **Default Value:** INFO
+- **Type:** string
+
+: Set the log level. One of `DEBUG`, `INFO`, `WARNING`, `ERROR`.
+
+###### APP_LABEL
+
+- **Required:** no
+    - **Default Value:** app.kubernetes.io/name
+- **Type:** string
+
+: Changes the label key used to identify applications.
+
+###### PROD_LABEL
+
+- **Required:** no
+    - **Default Value:** unset; matches all namespaces
+- **Type:** comma separated list of strings
+
+: Changes the namespace label key used to identify namespaces that are considered production environments.
+: **NOTE:** [PROD_LABEL](#prod_label) is ignored if [NAMESPACES](#namespaces) are provided
+
+###### NAMESPACES
+
+- **Required:** no
+    - **Default Value:** unset; scans all namespaces
+- **Type:** comma separated list of strings
+
+: Restricts the set of namespaces from which metrics will be collected. ex: `myapp-ns-dev,otherapp-ci`
```

Comment:
I would remove this example and create a final section with an detailed example (just telling that it will monitor namespaces blabal, of app_label blabla)

Reply:
ok added better example on the first section, so should be clear now how to use both PROD_LABEL and NAMESPACES